.github/workflows/update-attribute-dictionary-json.yml

```diff
@@ -0,0 +1,58 @@
+name: Update Attribute Dictionary JSON
+
+on:
+  workflow_dispatch:
+  schedule:
+    # 12pm every day
+    - cron: '0 12 * * *'
```

Comment:
i think this is UTC so it's technically 5am pdt i think? not that the time even matters but we might wanna update the comment